manifests/06_operator_cr.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Storage
```

Comment:
By removing `include.release.openshift.io/single-node-developer: "true"`, the storage operator will **never** be installed in SNO cluster in any cloud and thus probably break all StatefulSet tests in CI.

IMO, we should keep `include.release.openshift.io/single-node-developer: "true"` there, Storage will be installed by default, however, customers will have option to remove it if they need to trim down the cluster.

Reply:
hmm, I saw an example PR where it was removed but now can't find it.
I'll return this lines as well then

Reply:
Fixed. Do we need capability annotations for IBM-related resources? Are they still managed by CVO?

Reply:
Yes, we still need annotations for IBM-related resources.

Reply:
got it, i'll keep them